PartsAppTests/ExcelHelperTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace PartsAppTests
+{
+    [TestClass]
+    public class ExcelHelperTests
+    {
+        private readonly Func<string, string> _method = PartsApp.ExcelHelper.OperationsExcelHelper.GetValidExcelBookTitle;
+
+        [TestMethod]
+        public void GetValidExcelBookTitle_TitleIsNull_ThrowsArgumentException()
+        {
+            string title = null;
+            Assert.ThrowsException<ArgumentException>(() => _method(title));
+        }
+
+        [TestMethod]
+        public void GetValidExcelBookTitle_TitleIsEmpty_ThrowsArgumentException()
+        {
+            string title = string.Empty;
+            Assert.ThrowsException<ArgumentException>(() => _method(title));
+        }
+
+        [TestMethod]
+        public void GetValidExcelBookTitle_TitleIsValid_GetsOriginalTitle()
+        {
+            string title = "ИП Иванов_27-08-2024";
+            var result = _method(title);
+            Assert.AreEqual(title, result);
+        }
+
+        [TestMethod]
+        public void GetValidExcelBookTitle_TitleContainsSomeForbiddenChars_GetsValidatedTitle()
+        {
+            string title = "ООО \"Автодор\", безнал";
+            var result = _method(title);
+            Assert.AreEqual("ООО Автодор безнал", result);
+        }
```

Comment:
Unify these test cases. Use [DataRow] and add more than just 2 test cases

Reply:
done